src/pdm/cli/commands/run.py

```diff
@@ -216,12 +228,24 @@ def _run_task(
         kind, _, value, options = task
         shell = False
         if kind == "cmd":
-            if not isinstance(value, list):
-                value = shlex.split(str(value))
-            args = value + list(args)
+            interpolations = 0
```

Comment:
There seems no need to accumulate this value, no loop is involved, and it is only used in `kind == "cmd"` branch.

Reply:
Removed